nest-forms-backend/src/nases/nases.service.ts

```diff
@@ -560,37 +560,35 @@ export default class NasesService {
 
     this.checkAttachments(await this.filesService.areFormAttachmentsReady(id))
 
+    const shouldBumpJsonVersion =
+      !this.versioningEnabled ||
+      versionCompareBumpDuringSend({
+        currentVersion: form.jsonVersion,
+        latestVersion: formDefinition.jsonVersion,
+      })
+
     // TODO rework! this is super fragile and QUEUED state was not originally meant for this
     // just before sending to nases, update form state to QUEUED
     // if the operation takes long, this prevents repeated sending
     // sendToNasesAndUpdateState must never throw, and if it does not succeed we update the state back to DRAFT & set NASES_SEND_ERROR
     await this.formsService.updateForm(id, {
       state: FormState.QUEUED,
+      formSummary,
```

Comment:
I didn't want to attach `formSummary` to not fully sent forms (it can occur that `DRAFT` form will have summary, but it is OK for now)